dm/ctl/common/config.go

```diff
@@ -18,12 +18,17 @@ import (
 	"flag"
 	"fmt"
 	"net"
+	"time"
 
 	"github.com/BurntSushi/toml"
 	"github.com/pingcap/dm/pkg/utils"
 	"github.com/pingcap/errors"
 )
 
+var (
```

Comment:
for the constant variable, we can change them to `const` (we have not done that before, but we can do that later)